yesod-bin/AddHandler.hs

```diff
@@ -78,7 +79,11 @@ addHandlerFiles cabal (name, handlerFile) pattern methods = do
     modify cabal $ fixCabal name
```

Comment:
do you mean `exposed-modules`? My `yesod init` came stock without `HomeSpec` being added to the cabal file